core/src/storage2/collections/hashmap/mod.rs

```diff
@@ -165,6 +165,12 @@ where
         self.keys.len()
     }
 
+    /// Returns the number of key-value pairs stored in the hash map.
+    #[cfg(test)]
+    pub fn len_cached_entries(&self) -> u32 {
```

Comment:
```suggestion
    pub(crate) fn len_cached_entries(&self) -> u32 {
```

Reply:
just makes the intent a bit clearer even though not necessary since it is `#[cfg(test)]`